src/extensions/extensionInterface.ts

```diff
@@ -193,8 +193,88 @@ export const externalUnitMetricsReducer: Reducer<ExternalSortingUnitMetric[], Ex
 }
 ////////////////////////////////
 
+// Recording selection
```

Comment:
The recording selection includes selected electrode IDs, current timepoint, time range, and amplitude scale factor.